src/main/java/org/corfudb/cmdlets/corfu_layout.java

```diff
@@ -252,6 +271,49 @@
                         e.toString(),
                         "stack: " + ExceptionUtils.getStackTrace(e));
             }
+        } else if ((Boolean) opts.get("update_layout")) {
+            // This is not strictly a low-level primitive for the layout Paxos protocol.
+            // Rather, it's here at glue for QuickCheck testing of higher-level
+            // layout API exercise in the same spirit that layout_qc.erl tests
+            // the lower-level Paxos protocol.
+
+            LayoutView lv;
+            CorfuRuntime rt = configureRuntimeAddrPort(opts);
```

Comment:
Again, runtimes should not be created in a cmdlet which shouldn't be using a runtime...